db/clinic/routers/schedule.py

```
@@ -15,6 +16,15 @@
 def get_schedules(db: Session = Depends(get_db)):
     return schedule.get_all_schedules(db)
 
+@router.get("/filter", response_model=List[schemas.ScheduleShow])
+def get_schedules_by_doctors_and_start_time(start_time: datetime, doctor_ids: List[int] = Query(None), db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)